### PR TITLE
Add image tags

### PR DESCRIPTION
**Pull request by cpanlasigui**

### CODE REVIEW

**Comment by cpanlasigui** [comment]
[Issue 5: Image Tags](https://trello.com/c/AkFluCLG/264-issue-5-image-tags)

**Comment by cpanlasigui** [comment]
[Issue 5: Image Tags](https://trello.com/c/AkFluCLG/264-issue-5-image-tags)

File: test/models/image_test.rb [hunk 1]
```diff
@@ -15,4 +15,10 @@ class ImageTest < ActiveSupport::TestCase
     refute_predicate image, :valid?
     assert_equal ['Not an image url'], image.errors.messages[:url]
   end
+  test 'accepts tags' do
```
Comment (mickiehsu) [inline test/models/image_test.rb]: would be great if we have a blank line between tests.

File: app/controllers/images_controller.rb [hunk 2]
```diff
@@ -6,7 +6,8 @@ def new
   end
 
   def create
-    @image = Image.new(url: params[:image][:url])
+    @image = Image.new(image_params)
+    p image_params
```
Comment (mickiehsu) [inline app/controllers/images_controller.rb]: We probably don't want this. Let's remove it.

File: app/views/images/show.html.erb [hunk 3]
```diff
@@ -1 +1,5 @@
 <%= image_tag(@image.url, class: 'js-image') %>
+
+<% p 'PRINTING' %>
+<% p @image.tag_list %>
```
Comment (mickiehsu) [inline app/views/images/show.html.erb]: we probably also don't want these.

File: test/models/image_test.rb (hunk 1 above)
Comment (cpanlasigui) [reply test/models/image_test.rb]: Added

File: app/controllers/images_controller.rb (hunk 2 above)
Comment (cpanlasigui) [reply app/controllers/images_controller.rb]: Removed

File: app/views/images/show.html.erb (hunk 3 above)
Comment (cpanlasigui) [reply app/views/images/show.html.erb]: Removed

**Pull request merged**